supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/ActionCommands.scala

```diff
@@ -61,14 +76,26 @@ object ActionCommands {
   }
 
   def delocalizeFileOrDirectory(containerPath: Path, cloudPath: Path, contentType: Option[ContentType]) = {
-    s"if [[ -d ${containerPath.escape} ]]; then ${delocalizeDirectory(containerPath, cloudPath)}; else ${delocalizeFile(containerPath, cloudPath, contentType)}; fi"
+    s"if [[ -d ${containerPath.escape} ]]; then ${delocalizeDirectory(containerPath, cloudPath, contentType)}; else ${delocalizeFile(containerPath, cloudPath, contentType)}; fi"
   }
 
   def localizeDirectory(cloudPath: Path, containerPath: Path) = retry {
-    s"${containerPath |> makeContainerDirectory} && gsutil -m rsync -r ${cloudPath.escape} ${containerPath.escape}"
+    recoverRequesterPaysError(cloudPath) { flag =>
+      s"${containerPath |> makeContainerDirectory} && gsutil $flag -m rsync -r ${cloudPath.escape} ${containerPath.escape}"
+    }
   }
 
   def localizeFile(cloudPath: Path, containerPath: Path) = retry {
-    s"gsutil cp ${cloudPath.escape} ${containerPath.escape}"
+    recoverRequesterPaysError(cloudPath) { flag =>
+      s"gsutil $flag cp ${cloudPath.escape} ${containerPath.escape}"
+    }
+  }
+  
+  def recoverRequesterPaysError(path: Path)(f: String => String) = {
+    val withoutProject = ""
+    val withProject = s"-u ${path.projectId}"
+
+    s"""${f(withoutProject)} 2> gsutil_output.txt; RC_GSUTIL=$$?; if [[ "$$RC_GSUTIL" -eq 1 ]]; then
+       | grep "$BucketIsRequesterPaysErrorMessage" gsutil_output.txt && echo "Retrying with user project"; ${f(withProject)}; fi """.stripMargin
```

Comment:
Leaving a note: A proper `woodard-cli` via `woodard-impl-gcs` should be handle this type of retry internally.

Reply:
Yes please !